lib/glue/finding.rb

```diff
@@ -61,4 +61,31 @@ def to_json
     json
   end
 
+  # This is to fit a common JSON schema.
+  # See:  https://github.com/OWASP/off/blob/master/owasp.off.schema.json
+  def to_off
```

Comment:
Why is this not part of the reporter? I don't think this should be on the generic finding class